.gitignore

```diff
@@ -1,2 +1,4 @@
+yarn.lock
+node_modules
```

Comment:
`translated-content` doesn't have a `package.json`, so these two lines should not be added.

```suggestion
```

Reply:
@DmitrySkripunov In the mean-time, translated-content did get a `package.json`. Would you mind opening a separate PR for adding these lines to `.gitignore`?

Reply:
Never mind, there already is one: https://github.com/mdn/translated-content/pull/9269